python/xorbits/_mars/lib/groupby_wrapper.py

```diff
@@ -155,7 +177,7 @@ def shape(self):
 
     @property
     def _selected_obj(self):
-        return getattr(self.groupby_obj, "_selected_obj")
+        return getattr(self.groupby_obj, "_selected_obj", None)
```

Comment:
Is there any access of this property? I've searched for this property and there's only one match: https://github.com/xprobe-inc/xorbits/blob/c2d8cdd9e8c8ee60a4f242fdd553ffc9e185c832/python/xorbits/_mars/dataframe/groupby/sample.py#L62

But it seems the parameter `groupby` is always a pandas object, not a GroupByWrapper.

Reply:
There is no other place to call ``_selected_obj `` on the GPU, and it is written here just to be compatible with CUDF.